document/js-api/index.bs

```diff
@@ -1169,13 +1201,23 @@ Note: Exported Functions do not have a \[[Construct]] method and thus it is not
 
 <div algorithm>
   To <dfn>run a host function</dfn> from the JavaScript object |func|, type |functype|, and [=list=] of [=WebAssembly values=] |arguments|, perform the following steps:
+    1. Let [|parameters|] → [|resultTypes|] be |functype|.
+    1. Let |jsArguments| be the result of [=coerce WebAssembly arguments=](|parameters|,|arguments|)
+    1. Let |ret| be [=?=] [$Call$](|func|, undefined, |jsArguments|).
```

Comment:
I'm skeptical that `?` is doing the right thing here. It means if |func| throws, this algorithm will return a throw completion to the caller, which AFAIU is core wasm.

Reply:
refactored